app/components/ChartCountryMetricTrend/index.js

```diff
@@ -146,8 +146,8 @@ function ChartCountryMetricTrend({
 
   // dummy data to force the area plot from 0
   const dataForceYRange = [
-    { x: new Date(`${parseFloat(minYear) - 0.1}`).getTime(), y: 0 },
```

Comment:
@tmfrnz changing how these dates are generated fixes the problem on Firefox - but I'm not sure I understand why the min and max years were being created in this way to begin with? i.e. with `parseFloat` and a subtraction / addition before getting the timestamp from each of them. Perhaps this change would break something else in the app?

Looking through the git blame it looks like you worked on this last year sometime... any memories of what this code was addressing?

Reply:
@iainkirkpatrick great! 

the original intention of adding the "dataForceYRange" was to make sure the y-axis will start at 0 (`{ ... , y: 0}`) but I then also wanted to add some additional "padding" on the x-axis so that the first and last data points do not sit on the extreme horizontal ends of the chart as they do now.

Compare 170-fix-time-trend-presentation 
![image](https://user-images.githubusercontent.com/5811956/84670358-50f3e280-af26-11ea-8d89-ff02b174c26d.png)
with dev-v3.1
![image](https://user-images.githubusercontent.com/5811956/84670520-86003500-af26-11ea-885a-e15a48e90879.png)

I guess I could have set the padding somehow as an absolute value but thought it would be nicer using a fraction of dates as the padding would be larger when less data points (CPR) and smaller for more (ESR). 

Either way, with react-vis now not supported, it might be worth considering an alternative library to produce the trend charts

Reply:
@tmfrnz I see! Cool - I agree it looks nicer with padding :) I'll have a timeboxed-go at reimplenting the padding next time I work on this issue. Thanks for the explanation!

Reply:
Happy to confirm that the Firefox issue is fixed